openapi/composer/openapi.yml

```diff
@@ -0,0 +1,230 @@
+---
+openapi: 3.0.1
+info:
+  version: '1'
+  title: OSBuild Composer
+  description: Service to build and install images.
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+paths:
+  /openapi.json:
+    get:
+      summary: Get the OpenAPI JSON specification
+      tags:
+        - meta
+      responses:
+        '200':
+          description: Returns this document
```

Comment:
Is this mandatory in OpenAPI? The generated code has the spec embedded, but it does not auto-generate this handler, right? Just wondering what the story is, I don't mind it at all.

Reply:
That's a good question. I had assumed so, but now I cannot find the reference. @Gundersanne do you happen to know (I shamelessly based this on your work)? I'll drop it for now until we know for sure.